.env

```diff
@@ -0,0 +1 @@
+PORT=8080
```

Comment:
what is this? do you need a gitignore for this?

Reply:
Me being dumb, it isn't even using dotenv like the other projects are